ext/opentelemetry-exporter-cloud-monitoring/README.rst

```diff
@@ -0,0 +1,54 @@
+OpenTelemetry Cloud Monitoring Exporters
+========================================
+
+This library provides classes for exporting metrics data to Google Cloud Monitoring.
+
+Installation
+------------
+
+::
+
+    pip install opentelemetry-exporter-cloud-monitoring
+
+Usage
+-----
+
+.. code:: python
```

Comment:
If this is just the contents of `docs/examples/cloud_monitoring/basic_metrics.py`, you could use `literalinclude:: path/to/file.py` so the docs will stay up to date

Reply:
Very useful :)